test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -190,5 +190,22 @@ void testLiteInterpreterLoadOrigJit() {
   ASSERT_THROWS_WITH(_load_for_mobile(ss), "file not found");
 }
 
+void testLiteInterpreterWrongMethodName() {
+  script::Module m("m");
+  m.register_parameter("foo", torch::ones({}), false);
+  m.define(R"(
+    def add(self, x):
+      b = 4
+      return self.foo + x + b
+  )");
+  std::stringstream ss;
+  m._save_for_mobile(ss);
+  mobile::Module bc = _load_for_mobile(ss);
```

Comment:
If you have a lot of tests like this, it might be worth factoring it out into a `string -> Module` function.